node/pkg/watchers/evm/watcher.go

```diff
@@ -745,6 +845,65 @@ func canRetryGetBlockTime(err error) bool {
 	return exists
 }
 
+// verifyAndPublish validates a MessagePublication to ensure that it's safe. If so, it broadcasts the message. This function
```

Comment:
`evm/watcher.go` has gotten too big. Please move this stuff into something like `verifier.go` and `verifier_test.go`.

Reply:
OK, can do